BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -410,4 +402,57 @@ + (void)resetInstancesForTesting
     [_SDKClients removeAllObjects];
 }
 
+#pragma mark - API URLs
+
++ (NSString *)APIBaseURL
+{
+    if (staticAPIBaseURL.length == 0) {
+        staticAPIBaseURL = BOXDefaultAPIBaseURL;
+    }
+    return staticAPIBaseURL;
+}
+
++ (NSString *)APIAuthBaseURL
+{
+    if (staticAPIAuthBaseURL.length == 0) {
+        staticAPIAuthBaseURL = BOXDefaultAPIAuthBaseURL;
+    }
+    return staticAPIAuthBaseURL;
+}
+
++ (NSString *)APIUploadBaseURL
+{
+    if (staticAPIUploadBaseURL.length == 0) {
+        staticAPIUploadBaseURL = BOXDefaultAPIUploadBaseURL;
+    }
+    return staticAPIUploadBaseURL;
+}
+
++ (void)setAPIBaseURL:(NSString *)APIBaseURL
```

Comment:
I wanted it to be static for all clients, and in fact I intentionally added code to proactively set the URL on all existing session objects in memory. There would have to be a **very** compelling use-case to justify allowing the URLs to differ between client/session instances IMHO.